routesrv/polling.go

```diff
@@ -109,6 +110,12 @@ func (p *poller) poll(wg *sync.WaitGroup) {
 			}
 			span.SetTag("routes.count", routesCount)
 			span.SetTag("routes.bytes", routesBytes)
+
+			if updated && log.GetLevel() == log.DebugLevel {
```

Comment:
Maybe `>= log.DebugLevel` here? There is at least one more verbose leve: `TraceLevel`

https://pkg.go.dev/github.com/sirupsen/logrus#pkg-variables

Reply:
Indeed, fixed.